src/components/Table/Table.jsx

```diff
@@ -62,6 +62,7 @@ const propTypes = {
     hasResize: PropTypes.bool,
     /** If true removes the "table-layout: fixed" for resizable tables  */
     useAutoTableLayoutForResize: PropTypes.bool,
+    wrapCellText: PropTypes.bool,
```

Comment:
I think it would be clearer if we codified this into formal props
```
wrapCellText: oneOf['true', 'false', inherit]
``` 
or something similar, again with it defaulting to `true`.

Maybe `auto` is better than `inherit`?

```
wrapCellText: oneOf['true', 'false', auto]
```
Maybe `auto` implies too much though. Thoughts?

Reply:
I think this just ends up being a boolean that applies a class if true using the `classnames` package.

Reply:
Yes, that is way better! I think we can make the intent even more clear by using `PropTypes.oneOf(['always', 'never', 'auto'])`

Reply:
@davidicus there are 3 scenarios for the text here so we can use
'always' =  always use wrapping
'never'  = prevent wrapping and force wide cells
'auto' = truncate where possible and wrap if needed